lib/parser.js

```diff
@@ -593,7 +593,7 @@ class Parser {
 
   unknownWord(tokens) {
     throw this.input.error(
-      'Unknown word',
+      'Unexpected character "' + tokens[0][1] + '" in selector. Check CSS selector documentation for proper syntax.''Unknown word',
```

Comment:
```suggestion
      'Unknown word: ' + tokens[0][1],
```

The `unknownWord` is called in several instances, not all of them are related to selectors.

Keeping the message as it was and only adding a bit more debugging info seems sufficient to me.

Reply:
Many tools may use `if (error.message.includes('Unknown word')` so this change will be breaking changes.

So I agree that we need to use `'Unknown word: ' + tokens[0][1],`